### PR TITLE
new actions and enhanced variables

### PR DESCRIPTION
**Pull request by naili-m**
## New Action: Create Malicious File Indicator:

- Creates a Zero day Docx file as a file indicator
- Zero day means each run the resulting file has a unique hash code
- The action accepts email and URL parameters to be embedded within the file, users can then extract it via **File Content Extraction connector**

## New Action: Create Simulated Alert

- A dynamically generated Alert based on alert JSON definition, users can copy an alert defintion from FortiSOAR WebUI such as a response of 'GET /api/3/alerts/387dc349-b0c6-4317-a44c-0a83f7637cb5', replace some static entries such as timestamps, IP addresses, usernames ...etc with variables (Tags) so the action would replace these variables and then creates a new simulated alert

## Variables (Tags) now accept parameters

- Variables such as TR_RANDOM_INTEGER, can now be written as: TR_RANDOM_INTEGER,10,20 so the value of the variable will be a random # between 10 and 20, similarily TR_ASSET_IP can now take a network address so its value will be a random IP from that subnet, exp: TR_ASSET_IP, would yield: 192.168.100.X, X between 2,240

### CODE REVIEW

File: constants.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+LOGGER_NAME = 'soc_scenario'
```
Comment (cs-pooja-singh) [inline constants.py]: how is this different from fortisoar-soc-simulator/constants.py ?

File: fortisoar-soc-simulator/operations.py [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
 from connectors.core.connector import get_logger, ConnectorError
+from django.conf import settings
+from connectors.cyops_utilities.builtins import upload_file_to_cyops
+from docx import Document
+from io import BytesIO 
+from .constants import *
+from .utils import *
 
 logger = get_logger('FortiSOARSocSimulator')
+logger.setLevel(logging.DEBUG)
```
Comment (cs-pooja-singh) [inline fortisoar-soc-simulator/operations.py:12]: please remove before merge

File: constants.py (hunk 1 above)
Comment (cs-pooja-singh) [reply constants.py]: also, can we move the encoded file content to a separate file for better readability of other constants.

Comment (naili-m) [reply constants.py]: done in the latest commit

**Pull request merged**